internal/provisioner/registry/README.md

```diff
@@ -1,3 +1,278 @@
-# registry
+# Registry Provisioner
 
-This provisioner was removed in commit 76fefcfcd1408d04b4eb7eb368d3eaab49565237.
+## Summary
+
+The `registry` provisioner is a core Rukpak provisioner that knows how to interact with bundles of a particular format.
+These `registry+v1` bundles, or `registry` bundles, are container images containing a set of static Kubernetes YAML
+manifests organized in the legacy Operator Lifecycle Manger (OLM) format. For more information on the `registry+v1` format, see
+the [OLM packaging doc](https://docs.openshift.com/container-platform/4.10/operators/understanding/olm-packaging-format.html).
```

Comment:
nit: let's avoid any downstream documentation references in the upstream project. Could we point to https://olm.operatorframework.io/docs/tasks/creating-operator-manifests/ instead?